backend/src/environment.ts

```diff
@@ -87,6 +98,16 @@ export class Environment {
     this.manufacturerDetails.product.appCode = MANUFACTURE_ID_STANDARD
   }
 
+  checkOob() {
+    lan.checkForWifiNetwork(REMOTEIT_PI_WIFI, Environment.updateOob)
+  }
```

Comment:
Please move to LAN class and make promise.

Reply:
done